SECURITY.md

```diff
@@ -0,0 +1,4 @@
+# Security Policy
+
+Given that Servo does not yet have customers or products, we are comfortable accepting the security related vulnerabilities as a [new GitHub issue](https://github.com/servo/servo/issues/new) for now.
```

Comment:
Maybe mention [A-security](https://github.com/servo/servo/labels/A-security) label.

Reply:
I think externals contributors cannot set labels. One approach to fix this would be to have issues templates, as those can get labels assigned. We could have at least two of them: regular issue and security issue.

Maybe it's something we can do in a follow-up.